src/views/App/App.tsx

```diff
@@ -0,0 +1,56 @@
+import {displayName} from '@/../package.json';
+import {gameViewModel} from '@/services/gameViewModel';
+import {preferencesViewModel} from '@/services/preferencesViewModel';
+import {observer} from 'mobx-react';
+import {ActionButtons} from './ActionButtons/ActionButtons';
+import {Board} from './Board/Board';
+import {SolvedPuzzles} from './SolvedPuzzles/SolvedPuzzles';
+import {boardColorClass} from './boardColors/boardColors';
+
+import './boardColors/boardColors.css';
+
+const _App = () => {
```

Comment:
why do we need the underscore?
```suggestion
const App = observer( => {
```

Reply:
I am new to `mobx` library.
I separated them, since maybe fro testing purposes, we want to be able to render a plain component.
Maybe it is unnecessary. 
IDK, I saw spmeone on YouTube do it this way. 
If you have experience with it, please let me know how to make it better.